validator_client/src/duties_service.rs

```diff
@@ -137,8 +139,10 @@ impl<T: SlotClock + 'static, E: EthSpec> DutiesService<T, E> {
     pub fn block_proposers(&self, slot: Slot) -> HashSet<PublicKeyBytes> {
         let epoch = slot.epoch(E::slots_per_epoch());
 
-        // this is the set of local pubkeys that are not currently in a doppelganger detection period
-        let signing_pubkeys = self.validator_store.signing_pubkeys_hashset();
+        // Only collect validators that are not presently disabled for doppelganger protection.
```

Comment:
I think it's a little confusing to figure out what "not disabled for doppelganger protection" means

```suggestion
        // Only collect validators that are considered safe in terms of doppelganger protection.
```